src/PBot.Tests.Integration/Helper.cs

```diff
@@ -5,27 +5,35 @@
 //copy pasted from Octokit.net
 public static class Helper
 {
-    static readonly Lazy<Credentials> _credentialsThunk = new Lazy<Credentials>(() =>
```

Comment:
Sorry but readonly is of so little value especially when u take into account people applying it to mutable structures.

Also people often argue that "but what if i forget that a field should be readonly" if that is the case split up the class until the complexity is less and u wont need to remember which fields are readonly.

Also it doesnt really protect u against anything. i have actually watched a developer set a readonly field in code, the ide gave them a hint to make it not readonly and they selected it. it is a false protection. It operates under the assumption that in lack of knowledge (due to complexity) u will make the correct decision. and rather than removing the complexity people use readonly and enable complexity.

@adamralph  Have you ever been prevented from adding a bug by the readonly keyword? i know i havnt